docs/INSTALL.md

```diff
@@ -0,0 +1,88 @@
+# Installation
+
+## Docker Installation (recommended)
+
+Docker is a self-contained environment that can be used to run the code without having to install the dependencies.
+We have built a docker image with Ubuntu 20.04, ROS Noetic, and PyTorch 1.13.1.
+Make sure you have [nvidia-docker](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html) installed.
+At the root of the repository, run the following command to build the container:
+
+```bash
+docker compose build base
+```
+
+Run the container:
+
+```bash
+docker compose run --rm gpu
+```
+
+The base container can build the code without having access to a GPU, and is useful for exploring the structure of the project and environment.
+The actual code must be run from a container that has access to a GPU.
+
+The docker-compose file at the root of the repository is configured to mount the current directory into the container.
+This means that any changes you make to the code on your host machine will be reflected in the container.
+
+## Manual Installation
+
+The code was tested on Ubuntu 20.04 using Python 3.8 and [PyTorch](http://pytorch.org) v1.3.1.
+NVIDIA GPUs are needed for both training and testing.
+Reference the [Dockerfile](../docker/Dockerfile.noetic) for build instructions and dependencies.
```

Comment:
I think the Dockerfile is a clearer way of reproducing the environment, but I've left the manual installation instructions here with modifications for modern versions of pytorch.